devlib/utils/parse_aep.py

```diff
@@ -0,0 +1,528 @@
+#!/usr/bin/env python
+#    Copyright 2018 Linaro Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import getopt
+import subprocess
+import signal
+import serial
+import time
+import math
+
+class AEP_parser(object):
```

Comment:
No underscores in class names, acronyms should not be capitalised.  So should be `AepParser`